src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -282,6 +282,8 @@ private function renderRow(array $row, $cellFormat)
      * @param array  $row
      * @param int    $column
      * @param string $cellFormat
+     *
+     * @return string
```

Comment:
Not something we do except if this is really unexpected, but that's not the case here.